mrmustard/lab/abstract/state.py

```diff
@@ -479,9 +496,9 @@ def __truediv__(self, other):
     def _repr_markdown_(self):
         table = (
             f"#### {self.__class__.__qualname__}\n\n"
-            + "| Purity | Num modes | Bosonic size | Gaussian | Fock |\n"
-            + "| :----: | :----: | :----: | :----: | :----: |\n"
-            + f"| {(self.purity):.3f} | {self.num_modes} | {'1' if self.is_gaussian else 'N/A'} | {'✅' if self.is_gaussian else '❌'} | {'✅' if self._ket is not None or self._dm is not None else '❌'} |"
+            + "| Purity | Norm | Num modes | Bosonic size | Gaussian | Fock |\n"
+            + "| :----: | :----: | :----: | :----: | :----: | :----: |\n"
+            + f"| {self.purity :.3f} | {self.norm :.3f} | {self.num_modes} | {'1' if self.is_gaussian else 'N/A'} | {'✅' if self.is_gaussian else '❌'} | {'✅' if self._ket is not None or self._dm is not None else '❌'} |"
```

Comment:
When the norm or purity of the state is below the precision (three decimal places in this case) it will appear as if it is zero. Would it be better to use scientific notation?
```suggestion
            + f"| {self.purity :.2e} | {self.norm :.2e} | {self.num_modes} | {'1' if self.is_gaussian else 'N/A'} | {'✅' if self.is_gaussian else '❌'} | {'✅' if self._ket is not None or self._dm is not None else '❌'} |"
```